### PR TITLE
'Preview in Latex' was not working in ORA.

### PR DESCRIPTION
**Pull request by awaisdar001**
[TNL-1755] (https://openedx.atlassian.net/browse/TNL-1755)

Fixed `Preview in Latex` button in Open Response Assessments was not rendering.
Fixed by correcting the `jquery-selector`.

Wrote `bok_choy` acceptance test to for verification.

### CODE REVIEW

**Comment by awaisdar001** [comment]
@symbolist @adampalay @muzaffaryousaf 
 Please review

**Comment by adampalay** [comment]
:+1: works great

**Comment by muzaffaryousaf** [comment]
Other than minor tweaks :+1:

File: test/acceptance/pages.py [hunk 1]
```diff
@@ -51,7 +51,6 @@ def submit(self):
         with self.handle_alert():
             self.q(css=".action--submit").first.click()
 
-
```
Comment (muzaffaryousaf) [inline test/acceptance/pages.py]: We leave 2 lines before class. Can you please add the removed line again.

File: test/acceptance/pages.py [hunk 2]
```diff
@@ -75,6 +74,29 @@ def submit_response(self, response_text):
         self.submit()
         EmptyPromise(lambda: self.has_submitted, 'Response is completed').fulfill()
```
Comment (muzaffaryousaf) [inline test/acceptance/pages.py]: @awaisdar001 can we refactor this line to reflect the new selector. https://github.com/edx/edx-ora2/blob/aj/tnl1755-fix-latex-button-on-ORA/test/acceptance/pages.py#L73

**Comment by awaisdar001** [comment]
@muzaffaryousaf I have addressed your comments, Please have a look.

**Comment by muzaffaryousaf** [comment]
Thanks for addressing the comments. Test is looking good now.!

**Pull request merged**